problems/decorate/problem.md

```diff
@@ -0,0 +1,48 @@
+Decorations are a good way to mix in commonly used functionality with your Seneca
+instance, using: `decorate(name, handler)`
+
+Where:
+- `name`: The name you wish to call the decorated function or object.
+- `handler`: The handler the decorate seneca with. This can be a function or an object.
+
+For instance here we add a `stamp` function:
+
```

Comment:
"For instance here we add a stamp function that will print a time stamp when invoked"